<launch>
    <param name="/use_sim_time" value="true" />
    
    <arg name="urdf_path" default="$(find thewalkingdead)/config/walker.urdf" />

     <!-- webots -->
    <node type="start_webots.sh" name="webots" pkg="thewalkingdead" output="log"/>


    <node type="solver_server_node" name="solver_server_node" pkg="thewalkingdead" output="screen">
        <param name="walker_urdf_path" value="$(arg urdf_path)" />
    </node>

    <node type="task2.py" name="task2" pkg="thewalkingdead" output="screen">
    </node>

</launch>